lib/mediaserver.js

```diff
@@ -359,16 +359,24 @@ class MediaServer extends Emitter {
 
       // scenario 1: plain old RTP
       if (!requiresDtlsHandshake(req.body)) {
-        const endpoint = await this.createEndpoint({
+        this.createEndpoint({
```

Comment:
If you remove await and handle error in callback then caller is never going to know of error.  I would think it would be better to use await with try catch

Reply:
If error raised, callback is called like below.

          if(err) {
            return callback(err);
          }

then caller can know of this error even if caller does not set callback function because caller does not set, connectCaller function is handled with promise and rejected to caller.

This code has the same pattern of your code when with SRTP(else condition) and I think this is reasonable. 
How do you think?

Reply:
I changed to use try catch.

Thank you.